crap-code.kt

```diff
@@ -0,0 +1,26 @@
+package badcode.example
+class PerformanceFile {
+    val largeList = List(1000000) { it }
+    // Inefficient way to calculate the sum of elements
+    var sum = 0
+    fun calculateSum() {
+        for (i in 0 until largeList.size) {
+            sum += largeList[i]
```

Comment:
Use the sum() method below for better performance:
```suggestion
// Inefficient way to calculate the sum of elements
    var sum = 0
    fun calculateSum() {
        sum = largeList.sum()
        // typo in println() function
        println("Sum: sum")
        // Unnecessary object creation inside a loop
        for (i in 0 until 100000) {
            val tempObject = Any() // Creates a new object on each iteration
```